system-int-test/test/cmr/system_int_test/search/variable/concept_retrieval_test.clj

```diff
@@ -0,0 +1,109 @@
+(ns cmr.system-int-test.search.variable.concept-retrieval-test
+  "Integration test for variable retrieval via the /concepts/:concept-id and
+  /concepts/:concept-id/:revision-id endpoints."
+  (:require
+   [cheshire.core :as json]
```

Comment:
I'm still seeing a bunch of unused requires. Things like spatial search, all the various collection specific formats, etc will never be used for variables.